npm/angular/src/mount.ts

```diff
@@ -169,6 +171,22 @@ function bootstrapModule<T> (
   return testModuleMetaData
 }
 
+@Injectable()
+export class CypressTestComponentRenderer extends TestComponentRenderer {
+  override insertRootElement (rootElId: string) {
```

Comment:
Did not know about `override` 🤯

Reply:
Didn't need to use it here, but I wanted to follow the same pattern Angular is using for the `TestComponentRenderer`. Doubt I'll ever use it personally!